pkg/reconciler/configuration/resources/revision.go

```diff
@@ -40,8 +40,16 @@ func MakeRevision(config *v1alpha1.Configuration) *v1alpha1.Revision {
 
 	UpdateRevisionLabels(rev, config)
 
-	if rev.Annotations == nil {
-		rev.Annotations = make(map[string]string)
+	// Populate the CreatorAnnotation from configuration.
+	cans := config.GetAnnotations()
+	creator, ok := cans[serving.CreatorAnnotation]
+	if ok {
+		rans := rev.GetAnnotations()
+		if rans == nil {
+			rev.Annotations = map[string]string{serving.CreatorAnnotation: creator}
+		} else {
+			rans[serving.CreatorAnnotation] = creator
```

Comment:
I think we also need to call `rev.SetAnnotations(rans)` to make sure the annotation on rev is saved

Reply:
Thank you. Updated as same style of Service's implementation. (=call `SetAnnotations()` when setting initial annotation.)

https://github.com/knative/serving/blob/ebfa7145169a59a313b295c7f199786a45f46bcb/pkg/apis/serving/v1alpha1/service_defaults.go#L29